packages/dx-react-grid/docs/guides/basic-usage.md

```diff
@@ -0,0 +1,142 @@
+# Basic Usage
+
+## Overview
+
+The main purpose of React Grid is showing data passed into it. To specify data that Grid should display you can use the `rows` property.
+
+The next basic property of Data Grid is `columns`. It defines what row fields should be used by Grid.
+
+```js
+<Grid
+  rows={[
+    { region: 'South America', 'sector': 'Banking' },
+    { region: 'Africa', 'sector': 'Manufacturing' },
+    /* ... */
+  ]},
+  columns={[
+    { name: 'region', title: 'Region' },
+    { name: 'sector', title: 'Sector' },
+    /* ... */
+  ]},
+>
+</Grid>
+```
+
+Actually, the described properties give React Grid information about data.
+
+As for UI, there are two common plugins that allow representing records.
+
+The first one is the [TableView](../reference/table-view.md). It renders data as a simple table:
+
+
+```js
+<Grid
+  rows={rows},
+  columns={columns},
+>
+  <TableView />
+</Grid>
+```
+
+The second is the [TableHeaderRow](../reference/table-header-row.md) plugin. Pay attention, it should be placed after the `TableView`.
+
+```js
+<Grid
+  rows={rows},
+  columns={columns},
+>
+  <TableView />
+  <TableHeaderRow />
+</Grid>
+```
+
+The following demo shows the basic scenario of React Grid usage.
+
+.embedded-demo(basic/basic)
+
+## Basic Appearance Customization
+
+The `TableView` and `TableHeaderRow` plugins provide the rich API for appearance customization.
+
+Often, it's necessary to use more complex markup instead of plain text within a table cell. For example, in case if you are interested in implementing conditional formatting functionality.
+
+Let's look how to achieve it by using the `tableCellTemplate` property of the `TableView` plugin.+
+
+NOTE: the following code uses Bootstrap 3 templates. If you'd like to use Material UI, replace the `td` tag to `TableCell`.
```

Comment:
I think that providing a "broken example in some themes" is not a good decision. Our embedded demos already have an associated source code link.